ql/src/java/org/apache/hadoop/hive/ql/ddl/table/drop/DropTableAnalyzer.java

```diff
@@ -54,9 +55,11 @@ public void analyzeInternal(ASTNode root) throws SemanticException {
       outputs.add(new WriteEntity(table, WriteEntity.WriteType.DDL_EXCLUSIVE));
     }
 
+    TableName tName = TableName.fromString(tableName, null, null);
+
     boolean purge = (root.getFirstChildWithType(HiveParser.KW_PURGE) != null);
     ReplicationSpec replicationSpec = new ReplicationSpec(root);
-    DropTableDesc desc = new DropTableDesc(tableName, ifExists, purge, replicationSpec);
+    DropTableDesc desc = new DropTableDesc(tName, ifExists, purge, replicationSpec);
```

Comment:
why is this change? i do not see constructor that accepts TableName. https://github.com/apache/hive/blob/08fd61d70fd172a63ee03e52f8c11db4718ba43c/ql/src/java/org/apache/hadoop/hive/ql/ddl/table/drop/DropTableDesc.java#L41